modules/ROOT/pages/configure-headers4-task.adoc

```diff
@@ -1,37 +1,69 @@
 
-* `outboundHeadersMapName`
-+
-Supports sending custom headers with the response.
-+
-* `httpStatusVarName`
+== Configure Variable Names
+
+The APIkit Router configuration encompasses two specific properties that enable you to configure the variables that the HTTP listener searches for by default:
+
+* `outboundHeadersMapName`: Name of the variable that contains a headers map added in the response (`outboundHeaders` by default).
 +
-Supports sending a custom HTTP status code in the response.
+* `httpStatusVarName`: Name of the variable that stores the status code to be set in the response (`httpStatus` by default).
+
+Both variables are created by the APIkit Router and facilitate the inclusion of custom HTTP headers and status codes in the response.
+
+// TODO Maybe an image of the Router configuration as well?
+
+[source,xml,linenums]
+----
+<apikit:config  name="api-config"
+                api="api.raml"
+                outboundHeadersMapName="outboundHeaders"
+                httpStatusVarName="httpStatus"/>
+----
 
+If you opt to modify the default names of these variables, it is crucial to update the default HTTP listener configuration so to ensure alignment between the variable names and the configuration.
+
+[source,xml,linenums]
+----
+<http:listener config-ref="api-httpListenerConfig" path="/api/*">
+    <http:response
+        statusCode="#[vars.httpStatus default 200]">
+        <http:headers>#[vars.outboundHeaders default {}]</http:headers>
+    </http:response>
+    <http:error-response
+        statusCode="#[vars.httpStatus default 500]">
+        <http:body>#[payload]</http:body>
+        <http:headers>#[vars.outboundHeaders default {}]</http:headers>
+    </http:error-response>
+</http:listener>
+----
 
 == Add a Header to a Response
 
 Put the header you want to add inside the `outboundHeaders` map, as shown in the following example:
 
 [source,xml,linenums]
 ----
-<ee:transform>
-    <ee:set-variables>
-    <ee:set-variable name="outboundHeaders">
-        variables.outboundHeaders ++{headerName:"value"}
-    </ee:set-variable>
-</ee:set-variables>
-</ee:transform>
+<set-variable
+    value='#[vars.outboundHeaders default {} ++ {newHeaderName: "Header value"}]'
+    variableName="outboundHeaders"/>
 ----
 
+== Set the Status Code of a Response
 
-== See Also
+To assign a specific value to the `httpStatus` variable, you can use the following example as a reference:
 
+[source,xml,linenums]
+----
+<set-variable
+    value="201"
+    variableName="httpStatus"/>
+----
 
-* xref:apikit-simulate.adoc[To Simulate API Calls using API Console]
-* xref:validate-4-task.adoc[Validate Query Parameters and Headers Using APIkit]
\ No newline at end of file
+== See Also
+// TODO Fix the reference to the HTTP listener
+* https://docs.mulesoft.com/http-connector/1.7/http-listener-ref#sending-http-responses[Sending HTTP Responses when configuring HTTP listener source in Mule 4^]
```

Comment:
This text should be intial capped if it is a topic title.  Also, the carat is for use when we link outside of docs.mulesoft.com.

Reply:
OK! Thanks!